README-fr.md

```diff
@@ -32,6 +32,8 @@
 - [Des outils pour faciliter des ateliers en ligne par Carole Laimay](https://www.carole-laimay.com/outils-faciliter-ateliers-en-ligne/)
 - [Calendly : un outil simple pour organiser et programmer ses réunions](https://www.blogdumoderateur.com/calendly-outil-simple-organiser-programmer-rendez/)
 - [Bonnes pratiques en télétravail – #1 La gestion du temps](https://blog.zenika.com/2020/06/19/bonnes-pratiques-en-teletravail-1-la-gestion-du-temps/)
+![Bonnes pratiques en télétravail – #1 La gestion du temps | Infographie](https://media-exp1.licdn.com/dms/image/C4D22AQGObb6san-TJw/feedshare-shrink_2048_1536-alternative/0?e=1605744000&v=beta&t=UDMNzZnHGSo06aUOkvisbnasQjv7EffV2R5rg5m-Mc4)
```

Comment:
Hey @cycy-realbeats 👋 

Thanks for your contribution. 💚 
From my perpective, the image is too large to be in this README.
[See here](https://github.com/zenika-open-source/awesome-remote-work/blob/04563bc89d2d0244b50862db7641b08843585905/README-fr.md)

💡  Suggested solutions:

- Only add the image as a link
- Add the image at the end of the README
- Create a dedicated section or file to show pictures

What do you think? 👍